### PR TITLE
Ketabase | database connection

### PR DESCRIPTION
**Pull request by K1ngmar**
<3

### CODE REVIEW

**Comment by K1ngmar** [comment]
linter fail 🙃

**Comment by mraasvel** [comment]
Conventional PR message?

**Review by mraasvel — APPROVED** [review]
Clean up and some bug fixes, LGTM :^)

File: src/backend/app/src/main.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { NestFactory } from '@nestjs/core';
 import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
-import { AppModule } from './app.module';
+import { AppModule } from '../app.module';
```
[the remark this answers is not in the document]
Comment (K1ngmar) [reply src/backend/app/src/main.ts]: well if you say it like that ✅

**Review by Maqrkk — APPROVED** [review]
Ik vind het goed I guess

**Pull request merged**